packages/tools/pie-icons-configs/configs-vue.js

```diff
@@ -6,7 +6,7 @@ import {
 } from './configs.js'; // eslint-disable-line import/extensions
 
 /**
- * Object with iconSize props configuration for large and regular icons
+ * Object with size props configuration for large and regular icons
  */
 export const iconSize = {
```

Comment:
Should this be updated to `size`?

Reply:
I think not as this const is imported in vue package and makes sense to leave it as `iconSize` as more descriptive.